### DISCUSSION TITLE
Datepicker is not compatible with the a17-locale

### DISCUSSION THREAD

**Discussion by ermyril** [area17/twill#1129] [Ideas]
Hello!
I'm using twill 1.2.1

a17-datepicker appears to be incompatible with the a17-locale

```
      <a17-locale
        type="a17-datepicker"
        :attributes="{
                     label: 'Upload Date',
                     name: fieldName('uploadDate'),
                     }" >
      </a17-locale>
```

this code in the block template gives the following result

![image](https://user-images.githubusercontent.com/11752471/101748146-3b2e3700-3add-11eb-8f4a-8973fa3f92ab.png)

**Comment by ptrckvzn** [comment]
Hello @ermyril, thanks for opening the issue. Can you detail a bit more what you're trying to achieve? I assume you'd like to have the datepicker field translated?

**Comment by ermyril** [comment]
@ptrckvzn, Hi!
If I recall correctly I've had a form that had a multiple fields and all of them should have been translated i.e. have a different values for different languages, so I tried to use a17-locale wrapper for that purpose
